### ISSUE TITLE
Create dymanic elements 

### ISSUE THREAD

**Issue by ElioMS** [UniSharp/laravel-filemanager#196]
Hello,
It's possible to add the functionality to dynamic elements??
Creating with diferents data-input for div and different id for inputs but still not working
```
var count_flm = $('.ui.teal.button').length;
var new_count = count_flm +1;
                        html+= '<div class="form-group">'
			html+= '<label> Imagen </label>'
			html+= '<div class="ui fluid left action input">'
			html+= '<p class="title-image-flm"></p>'
			html+= '<div data-input="thumbnail'+new_count+'" class="ui teal button fm-button">  <i class="image icon"></i> Choose </div>'
			html+= '<input id="thumbnail'+new_count+'" class="input-flm" type="hidden" name="compraImg[]">'
			html+= '</div>'
			html+= '<label> Texto </label>'
			html+= '<input type="text" name="contenido[]" class="form-control" placeholder="Ingresa texto para el proceso de compra">'
			html+= '</div>'
```

Someone help me please? 
Thanks

**Comment by ElioMS** [comment]
Solved just add this to the script guys!

$(document).on('click', '.youridentifier' , function(e){
            localStorage.setItem('target_input', $(this).data('input'));
            localStorage.setItem('target_preview', $(this).data('preview'));
            window.open('/laravel-filemanager?type=' + type, 'FileManager', 'width=900,height=600');
            return false;
        })

**Comment by harryprasath** [comment]
Im getting this error on image select on the file manager while using the dynamic element can you give solution for this?

Uncaught TypeError: window.opener.SetUrl is not a function
    at useFile (laravel-filemanager?type=image:645)
    at HTMLDivElement.onclick (laravel-filemanager?type=image:1)

**Comment by g0110280** [comment]
Solved in `v2.0.0-alpha6`.

**Comment by HakerTag** [comment]
for Image

$(document).on('click', '.input-image' , function(e){
            var target_input = $('#' + $(this).data('input'));
            var target_preview = $('#' + $(this).data('preview'));
            window.open('/laravel-filemanager?type=Images', 'FileManager', 'width=900,height=600');
            window.SetUrl = function (items) {
                var file_path = items.map(function (item) {
                   return item.url;
                }).join(',');

                target_input.val('').val(file_path).trigger('change');
                // clear previous preview
                target_preview.html('');

                // set or change the preview image src
                items.forEach(function (item) {
                    target_preview.append(
                        $('<img>').css({ width: 100, height: 100 }).attr('src', item.thumb_url)
                    );
                });

                // trigger change event
                target_preview.trigger('change');

            }
	})


for Files

$(document).on('click', '.input-file' , function(e){
            var target_input = $('#' + $(this).data('input'));
            window.open('/laravel-filemanager?type=Files', 'FileManager', 'width=900,height=600');
            window.SetUrl = function (items) {
                var file_path = items.map(function (item) {
                   return item.url;
                }).join(',');

                target_input.val('').val(file_path).trigger('change');
            }
	})
